support-code-examples.html

```diff
@@ -68,60 +68,174 @@
         });
       });
     }
-                            let timesPressed = 0;
-    document.addEventListener("keydown", function (e) {
-      if (e.ctrlKey && e.key == "Enter") {
-        showSessionCode();
-      }
-      if (e.key === 'n') {
-        timesPressed++;
-        if (timesPressed === 3) {
-          timesPressed = 0;
-          showSessionCode();
-        }
-      } else {
-        timesPressed = 0;
-      }
-    });
+  </script>
 
-    document.addEventListener('touchstart', (event) => {
-      if (event.touches.length == 3) {
-        showSessionCode();
-      }
-    });
+  <style type="text/css">
```

Comment:
Nit pick but I would recommend putting the styles at the bottom so the first thing people see is the code they need to understand.

Reply:
Or perhaps we add common styles to a shared css file so all examples have the standard style.